app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -19,19 +20,25 @@ const StatusUpdateSection = () => (
   </ContentSection.Content>
 )
 
-const LeaseInformationSection = ({ statusHistory }) => (
-  <ContentSection title='Lease Information'>
-    <ContentSection.Sub title='Demographics'>
-      <DemographicsInputs />
-    </ContentSection.Sub>
-    {!isEmpty(statusHistory) && (
-      <ContentSection.Sub title='Status History' borderBottom={false}>
-        <StatusList items={statusHistory} onAddCommnent={() => window.alert('add comment')} />
+const LeaseInformationSection = withContext(({ store }) => {
+  const { statusHistory } = store
+  return (
+    <ContentSection title='Lease Information'>
+      <ContentSection.Content borderBottom>
+        <LeaseInformatonInputs />
+      </ContentSection.Content>
+      <ContentSection.Sub title='Demographics'>
+        <DemographicsInputs />
       </ContentSection.Sub>
-    )
-    }
-  </ContentSection>
-)
+      {!isEmpty(statusHistory) && (
+        <ContentSection.Sub title='Status History' borderBottom={false}>
+          <StatusList items={statusHistory} onAddCommnent={() => alert('add comment')} />
```

Comment:
does this status piece really belong to the lease information section?

Reply:
don't lose this comment among all the other ones!

Reply:
I cannot find the invisionapp where I got this from. I see what you are saying. It does not feel the status history is part of the lease. I structured it this way because in the invision screen this seems to be under the sub header (h2) "Lease Information" and the status history was clearly a h3 sub header.  We could ask jesse to get more info.

Reply:
@slowbot is the StatusList meant to fall under the Lease Information Section?

Reply:
update from Jesse: Break it into it's own section.